asmcli/lib/context.sh

```diff
@@ -64,7 +64,8 @@ context_init() {
     "OFFLINE": "${OFFLINE:-0}",
     "INCLUDES_PROMETHEUS": "${INCLUDES_PROMETHEUS:-0}",
     "INCLUDES_STACKDRIVER": "${INCLUDES_STACKDRIVER:-1}",
-    "CHANNEL": "${CHANNEL:-}"
+    "CHANNEL": "${CHANNEL:-}",
+    "OPENSHIFT_CLUSTER_SETUP": "${OPENSHIFT_CLUSTER_SETUP:-}"
```

Comment:
Can we name this variable something more general? e.g. `PRIVATE_ISSUER` or `USE_JWKS_NOT_OIDC` or `NO_PUBLIC_ISSUER` or something. It doesn't fundamentally have anything to do with Openshift.

Reply:
yeah sounds good, I used that name because it is only being used for openshift now. Good suggestion, will make the change.